backend/app/tests/core/test_security.py

```
@@ -0,0 +1,66 @@
+# IMPORTANT: This is a fixed key for predictable tests.
+# In a real scenario, the key from settings would be used.
+TEST_FERNET_KEY = Fernet.generate_key()
+TEST_FERNET_KEY_STR = TEST_FERNET_KEY.decode('utf-8')
+
+@pytest.fixture
+def mock_settings_key():
+    # Patch settings.APP_SYMMETRIC_ENCRYPTION_KEY to use our test key
+    with patch.object(settings, 'APP_SYMMETRIC_ENCRYPTION_KEY', TEST_FERNET_KEY_STR) as _fixture:
+        yield _fixture
+
+def test_decrypt_password_success(mock_settings_key):
```

Ruff (0.11.9)

Unused function argument: `mock_settings_key`

(ARG001)